imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -1109,6 +1113,8 @@ public void cdrProcessPhase5(MotechEvent event) {
             return;
         }
 
+        eventRelay.broadcastEventMessage(new MotechEvent(CSR_VERIFIER_CACHE_EVICT_MESSAGE));
```

Comment:
No need, it takes quite a while to queue up all the CSR chunk dispatch events.
The message will be broadcast and processed way before the first dispatched chunk is being processed.